### DISCUSSION TITLE
Remove unnecessary color CSS vars

### DISCUSSION THREAD

**Discussion by xndyz** [chakra-ui/chakra-ui#4521] [General]
![image](https://user-images.githubusercontent.com/19108762/128271665-37708c27-ae62-44bf-9ba3-b9eb1795a930.png)

Chakra currently creates a bunch of unused variables for social apps. Is there any way to remove this if we don't need it?

**Comment by aaronkik** [comment]
I haven't tried this myself yet, but checkout the answer in this thread. https://github.com/chakra-ui/chakra-ui/discussions/4275#discussioncomment-911378
